### DISCUSSION TITLE
Upgrade graceful-fs@3.0.8 TO graceful-fs@4.0.0

### DISCUSSION THREAD

**Discussion by Aukhan** [exceljs/exceljs#2534] [Ideas & Proposals]
Hi Guys,

Awesome package put together here,

geaceful-fs 3 and below throws a breakage/deprecated warning with node v6.0.0 and needs to be upgraded to graceful-fs@4.0.0

Thanks

**Comment by dcrystalj** [comment]
please make pr

**Comment by UziTech** [comment]
unzip2 needs to update their dependancies first then exceljs will need to update its version of unzip2

https://github.com/glebdmitriew/node-unzip-2/issues/13

**Comment by apydo** [comment]
This issue is a duplicate of #79.

unzip2 is unmaintained for 2 years now.

**Comment by Aukhan** [comment]
@apydo thanks
Is there a way out of this ?
Doesn't seem like unzip2 is getting any attention ?

From the git repo it seems like unzip2 was created to support STORE compression and was forked from node-unzip 
https://github.com/EvanOxfeld/node-unzip

So, is there a way out of this ?
This package certainly needs to move forward as a lot of us have work depending on it.

Could we consider refactoring to node-zip
https://github.com/daraosn/node-zip
